src/patternfly/components/Tabs/examples/Tabs.md

```diff
@@ -194,3 +204,4 @@ Whenever a list of tabs is unique on the current page, it can be used in a `<nav
 | `.pf-m-fill` | `.pf-c-tabs` | Modifies the tabs to fill the available space. |
 | `.pf-m-current` | `.pf-c-tabs__item` | Indicates that a tab item is currently selected. |
 | `.pf-m-inset-{none, sm, md, lg, xl, 2xl}{-on-[md, lg, xl, 2xl]}` | `.pf-c-tabs` | Modifies the tabs component padding/inset to visually match padding of other adjacent components. |
+| `.pf-m-current-tab-background--light-300` | `.pf-c-tabs` | Modifies the tabs component tab background colors. |
```

Comment:
what if this was called pf-m-inverse instead ? since it just swaps the colors around ? I would also update the description

Reply:
I think @mceledonia had recommended this naming. The issue is that it doesn't scale if we offer a different variation, but it could work if we don't think this color change is part of a theme where we might have other color/theme variations.